ctwrap/simulation.py

```diff
@@ -471,12 +543,19 @@ def run_parallel(self,
         """
         Run variation using multiprocessing.
 
+        The :meth:`run_parallel` method runs all the variations in the input
+        file in parallel using python multiprocessing. A minimal example is:
+
+         .. code-block:: Python
+
+           # Runs all the variations in parallel
+           sh.run_parallel(sim)
```

Comment:
whenever you use a variable name (like `sh` here), make sure that it is mentioned in the text describing the example.